### PR TITLE
fixed nickname issue , now nickname is optional

### PR DESCRIPTION
**Pull request by Ac-Srikanth**

### CODE REVIEW

**Comment by ashutosh1919** [comment]
@Ac-Srikanth , I think the good solution would be to have variable `isNickName` in `setting` in `portfolio.js` file which you need to pass till greeting and only render the nickname component if the `isNickName` is `true`. Also please remove the changes in package-lock.json while committing the work.

**Comment by ashutosh1919** [comment]
You can see deployment of your PR here: https://deploy-preview-34--ashutosh1919portfolio.netlify.app/

**Comment by Ac-Srikanth** [comment]
@ashutosh1919 yes okay i will make those changes

**Comment by ashutosh1919** [comment]
> @ashutosh1919 yes okay i will make those changes, but should it be like the way you have done in main.js file for splash ?

Yes, you need to do it like that.

File: src/containers/greeting/Greeting.js [hunk 1]
```diff
@@ -1,56 +1,104 @@
...
-								<Button
-									text="⭐ Star Me On Github"
-									newTab={true}
-									href={greeting.portfolio_repository}
-									theme={theme}
-									className="portfolio-repo-btn"
-								/>
-							</div>
-							{/* <div className="button-greeting-div">
+  if (settings.isNickName) {
```
Comment (ashutosh1919) [inline src/containers/greeting/Greeting.js:11]: I think it's not a good idea to repeat the whole code. We can do like below.
Build seperate functional component which take isNickName as prop and give result as <h2> containing nickname. If isNickName is false then return empty <></>

Comment (Ac-Srikanth) [reply src/containers/greeting/Greeting.js:11]: Got it , i will try that.

**Comment by dineshnadimpalli** [comment]
Hi @Ac-Srikanth, as the code has changed since the last commit to this PR, can you resolve the conflicts and make a commit?

**Comment by Ac-Srikanth** [comment]
sorry for being this late , but i have made the changes.

**Comment by saiteja13427** [comment]
I think this is an overkill for the nickname thing,

Also, it has conflicts and is an old PR.

So, closing it. I will raise a new PR for this.

**Pull request closed without merging**